### PR TITLE
Add Bakeneko 60 keyboard

### PR DESCRIPTION
**Pull request by kkatano**
## Types of Changes

- [ ] Core
- [ ] Bugfix
- [ ] New feature
- [ ] Enhancement/optimization
- [x] Keyboard (addition or update)
- [ ] Keymap/layout/userspace (addition or update)
- [ ] Documentation

## Issues Fixed or Closed by This PR

* 

## Checklist

- [x] My code follows the code style of this project: [**C**](https://docs.qmk.fm/#/coding_conventions_c), [**Python**](https://docs.qmk.fm/#/coding_conventions_python)
- [x] I have read the [**PR Checklist** document](https://docs.qmk.fm/#/pr_checklist) and have made the appropriate changes.
- [ ] My change requires a change to the documentation.
- [ ] I have updated the documentation accordingly.
- [x] I have read the [**CONTRIBUTING** document](https://docs.qmk.fm/#/contributing).
- [ ] I have added tests to cover my changes.
- [x] I have tested the changes and verified that they work and don't break anything (as well as I can manage).

### CODE REVIEW

**Review by tzarc — APPROVED** [review]

**Review by fauxpark — APPROVED** [review]

File: keyboards/bakeneko60/config.h [hunk 1]
```diff
@@ -0,0 +1,146 @@
...
+//#    define RGBLIGHT_EFFECT_RGB_TEST
+//#    define RGBLIGHT_EFFECT_ALTERNATING
+/*== customize breathing effect ==*/
+/*==== (DEFAULT) use fixed table instead of exp() and sin() ====*/
+//#    define RGBLIGHT_BREATHE_TABLE_SIZE 256      // 256(default) or 128 or 64
+/*==== use exp() and sin() ====*/
+//#    define RGBLIGHT_EFFECT_BREATHE_CENTER 1.85  // 1 to 2.7
+//#    define RGBLIGHT_EFFECT_BREATHE_MAX    255   // 0 to 255
+//#endif
+
```
Comment (noroadsleft) [inline keyboards/bakeneko60/config.h:86]:
```suggestion
```

File: keyboards/bakeneko60/bakeneko60.h [hunk 2]
```diff
@@ -0,0 +1,61 @@
...
+ *
+ * You should have received a copy of the GNU General Public License
+ * along with this program.  If not, see <http://www.gnu.org/licenses/>.
+ */
+
+#pragma once
+
+#include "quantum.h"
+
+#define LAYOUT_all( \
```
Comment (noroadsleft) [inline keyboards/bakeneko60/bakeneko60.h:21]: This layout matches the [`60_ansi_split_bs_rshift` Community Layout](https://github.com/qmk/qmk_firmware/tree/4701001524276638992d1ae92b638ca68fd0c410/layouts/default/60_ansi_split_bs_rshift).

```suggestion
#define LAYOUT_60_ansi_split_bs_rshift( \
```

File: keyboards/bakeneko60/bakeneko60.h [hunk 3]
```diff
@@ -0,0 +1,61 @@
...
+    k400, k401, k402,                   k406,                         k410, k411, k412, k413  \
+) { \
+    { k000,  k001,  k002,  k003,  k004,  k005,  k006,  k007,  k008,  k009,  k010,  k011,  k012,  k013,  k014 }, \
+    { k100,  k101,  k102,  k103,  k104,  k105,  k106,  k107,  k108,  k109,  k110,  k111,  k112,  k113,  KC_NO}, \
+    { k200,  k201,  k202,  k203,  k204,  k205,  k206,  k207,  k208,  k209,  k210,  k211,  KC_NO, k213,  KC_NO}, \
+    { k300,  KC_NO, k302,  k303,  k304,  k305,  k306,  k307,  k308,  k309,  k310,  k311,  k312,  k313,  KC_NO}, \
+    { k400,  k401,  k402,  KC_NO, KC_NO, KC_NO, k406,  KC_NO, KC_NO, KC_NO, k410,  k411,  k412,  k413,  KC_NO}  \
+}
+
+#define LAYOUT_ansi( \
```
Comment (noroadsleft) [inline keyboards/bakeneko60/bakeneko60.h:35]:
```suggestion
#define LAYOUT_60_ansi( \
```

File: keyboards/bakeneko60/bakeneko60.h [hunk 4]
```diff
@@ -0,0 +1,61 @@
...
+    k400, k401, k402,                   k406,                         k410, k411, k412, k413  \
+) { \
+    { k000,  k001,  k002,  k003,  k004,  k005,  k006,  k007,  k008,  k009,  k010,  k011,  k012,  KC_NO, k014 }, \
+    { k100,  k101,  k102,  k103,  k104,  k105,  k106,  k107,  k108,  k109,  k110,  k111,  k112,  k113,  KC_NO}, \
+    { k200,  k201,  k202,  k203,  k204,  k205,  k206,  k207,  k208,  k209,  k210,  k211,  KC_NO, k213,  KC_NO}, \
+    { k300,  KC_NO, k302,  k303,  k304,  k305,  k306,  k307,  k308,  k309,  k310,  k311,  k312,  KC_NO, KC_NO}, \
+    { k400,  k401,  k402,  KC_NO, KC_NO, KC_NO, k406,  KC_NO, KC_NO, KC_NO, k410,  k411,  k412,  k413,  KC_NO}  \
+}
+
+#define LAYOUT_tsangan_hhkb( \
```
Comment (noroadsleft) [inline keyboards/bakeneko60/bakeneko60.h:49]:
```suggestion
#define LAYOUT_60_tsangan_hhkb( \
```

File: keyboards/bakeneko60/info.json [hunk 5]
```diff
@@ -0,0 +1,219 @@
+{
+    "keyboard_name": "Bakeneko 60",
+    "url": "https://github.com/kkatano/bakeneko-60-pcb",
+    "maintainer": "kkatano",
+    "width": 15,
+    "height": 5,
+    "layouts": {
+        "LAYOUT_all": {
```
Comment (noroadsleft) [inline keyboards/bakeneko60/info.json:8]:
```suggestion
        "LAYOUT_60_ansi_split_bs_rshift": {
```

File: keyboards/bakeneko60/info.json [hunk 6]
```diff
@@ -0,0 +1,219 @@
...
+                { "x":1.25, "y":4, "w":1.25 },
+                { "x":2.5, "y":4, "w":1.25 },
+                { "x":3.75, "y":4, "w":6.25 },
+                { "x":10, "y":4, "w":1.25 },
+                { "x":11.25, "y":4, "w":1.25 },
+                { "x":12.5, "y":4, "w":1.25 },
+                { "x":13.75, "y":4, "w":1.25 }
+            ]
+        },
+        "LAYOUT_ansi": {
```
Comment (noroadsleft) [inline keyboards/bakeneko60/info.json:79]:
```suggestion
        "LAYOUT_60_ansi": {
```

File: keyboards/bakeneko60/info.json [hunk 7]
```diff
@@ -0,0 +1,219 @@
...
+                { "x":1.25, "y":4, "w":1.25 },
+                { "x":2.5, "y":4, "w":1.25 },
+                { "x":3.75, "y":4, "w":6.25 },
+                { "x":10, "y":4, "w":1.25 },
+                { "x":11.25, "y":4, "w":1.25 },
+                { "x":12.5, "y":4, "w":1.25 },
+                { "x":13.75, "y":4, "w":1.25 }
+            ]
+        },
+        "LAYOUT_tsangan_hhkb": {
```
Comment (noroadsleft) [inline keyboards/bakeneko60/info.json:148]:
```suggestion
        "LAYOUT_60_tsangan_hhkb": {
```

File: keyboards/bakeneko60/keymaps/default/keymap.c [hunk 8]
```diff
@@ -0,0 +1,38 @@
...
+ */
+#include QMK_KEYBOARD_H
+
+enum layer_names {
+    _BASE,
+    _FN
+};
+
+const uint16_t PROGMEM keymaps[][MATRIX_ROWS][MATRIX_COLS] = {
+    [_BASE] = LAYOUT_all(
```
Comment (noroadsleft) [inline keyboards/bakeneko60/keymaps/default/keymap.c:24]:
```suggestion
    [_BASE] = LAYOUT_60_ansi_split_bs_rshift(
```

File: keyboards/bakeneko60/keymaps/default/keymap.c [hunk 9]
```diff
@@ -0,0 +1,38 @@
...
+        KC_ESC,  KC_1,    KC_2,    KC_3,    KC_4,    KC_5,    KC_6,    KC_7,    KC_8,    KC_9,    KC_0,    KC_MINS, KC_EQL,  KC_GRV,  KC_BSPC,
+        KC_TAB,  KC_Q,    KC_W,    KC_E,    KC_R,    KC_T,    KC_Y,    KC_U,    KC_I,    KC_O,    KC_P,    KC_LBRC, KC_RBRC,          KC_BSLS,
+        KC_CAPS, KC_A,    KC_S,    KC_D,    KC_F,    KC_G,    KC_H,    KC_J,    KC_K,    KC_L,    KC_SCLN, KC_QUOT,                   KC_ENT,
+        KC_LSFT,          KC_Z,    KC_X,    KC_C,    KC_V,    KC_B,    KC_N,    KC_M,    KC_COMM, KC_DOT,  KC_SLSH,          KC_RSFT, MO(_FN),
+        KC_LCTL, KC_LGUI, KC_LALT,                            KC_SPC,                                      KC_RALT, KC_RGUI, KC_APP,  KC_RCTL
+    ),
+    [_FN] = LAYOUT_all(
```
Comment (noroadsleft) [inline keyboards/bakeneko60/keymaps/default/keymap.c:31]:
```suggestion
    [_FN] = LAYOUT_60_ansi_split_bs_rshift(
```

File: keyboards/bakeneko60/keymaps/via/keymap.c [hunk 10]
```diff
@@ -0,0 +1,54 @@
...
+
+enum layer_names {
+    _L0,
+    _L1,
+    _L2,
+    _L3
+};
+
+const uint16_t PROGMEM keymaps[][MATRIX_ROWS][MATRIX_COLS] = {
+    [_L0] = LAYOUT_all(
```
Comment (noroadsleft) [inline keyboards/bakeneko60/keymaps/via/keymap.c:26]:
```suggestion
    [_L0] = LAYOUT_60_ansi_split_bs_rshift(
```

File: keyboards/bakeneko60/keymaps/via/keymap.c [hunk 11]
```diff
@@ -0,0 +1,54 @@
...
+        KC_ESC,  KC_1,    KC_2,    KC_3,    KC_4,    KC_5,    KC_6,    KC_7,    KC_8,    KC_9,    KC_0,    KC_MINS, KC_EQL,  KC_GRV,  KC_BSPC,
+        KC_TAB,  KC_Q,    KC_W,    KC_E,    KC_R,    KC_T,    KC_Y,    KC_U,    KC_I,    KC_O,    KC_P,    KC_LBRC, KC_RBRC,          KC_BSLS,
+        KC_CAPS, KC_A,    KC_S,    KC_D,    KC_F,    KC_G,    KC_H,    KC_J,    KC_K,    KC_L,    KC_SCLN, KC_QUOT,                   KC_ENT,
+        KC_LSFT,          KC_Z,    KC_X,    KC_C,    KC_V,    KC_B,    KC_N,    KC_M,    KC_COMM, KC_DOT,  KC_SLSH,          KC_RSFT, MO(_L1),
+        KC_LCTL, KC_LGUI, KC_LALT,                            KC_SPC,                                      KC_RALT, KC_RGUI, KC_APP,  KC_RCTL
+    ),
+    [_L1] = LAYOUT_all(
```
Comment (noroadsleft) [inline keyboards/bakeneko60/keymaps/via/keymap.c:33]:
```suggestion
    [_L1] = LAYOUT_60_ansi_split_bs_rshift(
```

File: keyboards/bakeneko60/keymaps/via/keymap.c [hunk 12]
```diff
@@ -0,0 +1,54 @@
...
+        RESET,   KC_F1,   KC_F2,   KC_F3,   KC_F4,   KC_F5,   KC_F6,   KC_F7,   KC_F8,   KC_F9,   KC_F10,  KC_F11,  KC_F12,  _______, _______,
+        _______, _______, _______, _______, _______, _______, _______, _______, _______, _______, _______, _______, _______,          _______,
+        _______, _______, _______, _______, _______, _______, _______, _______, _______, _______, _______, _______,                   _______,
+        _______,          _______, _______, _______, _______, _______, _______, _______, _______, _______, _______,          _______, _______,
+        _______, _______, _______,                            _______,                                     _______, _______, _______, _______
+    ),
+    [_L2] = LAYOUT_all(
```
Comment (noroadsleft) [inline keyboards/bakeneko60/keymaps/via/keymap.c:40]:
```suggestion
    [_L2] = LAYOUT_60_ansi_split_bs_rshift(
```

File: keyboards/bakeneko60/keymaps/via/keymap.c [hunk 13]
```diff
@@ -0,0 +1,54 @@
...
+        _______, _______, _______, _______, _______, _______, _______, _______, _______, _______, _______, _______, _______, _______, _______,
+        _______, _______, _______, _______, _______, _______, _______, _______, _______, _______, _______, _______, _______,          _______,
+        _______, _______, _______, _______, _______, _______, _______, _______, _______, _______, _______, _______,                   _______,
+        _______,          _______, _______, _______, _______, _______, _______, _______, _______, _______, _______,          _______, _______,
+        _______, _______, _______,                            _______,                                     _______, _______, _______, _______
+    ),
+    [_L3] = LAYOUT_all(
```
Comment (noroadsleft) [inline keyboards/bakeneko60/keymaps/via/keymap.c:47]:
```suggestion
    [_L3] = LAYOUT_60_ansi_split_bs_rshift(
```

File: keyboards/bakeneko60/rules.mk [hunk 14]
```diff
@@ -0,0 +1,22 @@
...
+CONSOLE_ENABLE = no         # Console for debug
+COMMAND_ENABLE = no         # Commands for debug and configuration
+# Do not enable SLEEP_LED_ENABLE. it uses the same timer as BACKLIGHT_ENABLE
+SLEEP_LED_ENABLE = no       # Breathing sleep LED during USB suspend
+# if this doesn't work, see here: https://github.com/tmk/tmk_keyboard/wiki/FAQ#nkro-doesnt-work
+NKRO_ENABLE = no            # USB Nkey Rollover
+BACKLIGHT_ENABLE = no       # Enable keyboard backlight functionality
+RGBLIGHT_ENABLE = no        # Enable keyboard RGB underglow
+BLUETOOTH_ENABLE = no       # Enable Bluetooth
+AUDIO_ENABLE = no           # Audio output
```
Comment (noroadsleft) [inline keyboards/bakeneko60/rules.mk:22]:
```suggestion
AUDIO_ENABLE = no           # Audio output

LAYOUTS = 60_ansi_split_bs_rshift 60_ansi 60_tsangan_hhkb
```

**Comment by noroadsleft** [comment]
Thanks!

**Pull request merged**